.github/workflows/pullrequest.yml

```
@@ -204,26 +204,45 @@
-        if: ${{ github.event_name == 'pull_request' }}
-        with:
-          token: ${{ secrets.DIRTY_FIX_BOT_TOKEN }}
-          ref: ${{ github.event.pull_request.head.ref }}
 
       - name: Setup yarn
         uses: ./.github/actions/setup-yarn
-
-      - name: load-deps
-        uses: ./.github/actions/load-deps
+        with:
+          RUNS_ON_S3_BUCKET_CACHE: ${{ vars.RUNS_ON_S3_BUCKET_CACHE }}
```

actionlint (1.7.7)

input "RUNS_ON_S3_BUCKET_CACHE" is not defined in action "Setup yarn" defined at "./.github/actions/setup-yarn". available inputs are "working-directory"

(action)